.travis.yml

```diff
@@ -9,6 +9,10 @@ os:
   - linux
   - osx
 
+env:
+  global:
+    - JULIA_PKG_SERVER=""
```

Comment:
Adding this globally feels like a bit of a problem since the whole Julia ecosystem is running with this on (in 1.5).

Reply:
I don't have a strong opinion about this other than that it prevented meaningful CI results at the time. I can back it out.

Reply:
Very funny. With a `git reset`, Travis just reported the old results for that commit.